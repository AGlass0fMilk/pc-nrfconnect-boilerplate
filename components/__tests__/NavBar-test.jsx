import React from 'react';
import renderer from 'react-test-renderer';
import {
    decorateNavBar,
} from '../NavBar';

describe('NavBar', () => {
    it('should implement component decorator', () => {
        expect(decorateNavBar).toBeInstanceOf(Function);
    });
});

describe('decorateNavBar', () => {
    it('should render component', () => {
        const NavBar = decorateNavBar('div');
        expect(renderer.create(
            <NavBar />,
        )).toMatchSnapshot();
    });
});
