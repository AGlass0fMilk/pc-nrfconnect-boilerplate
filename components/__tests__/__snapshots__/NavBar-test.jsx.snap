exports[`decorateNavBar should render component 1`] = `<div />`;
